sml_builder/templates/help_category.html

```diff
@@ -17,18 +16,116 @@
+        }}
+      <p>Method requests can be submitted to the ONS Statistical Methods Library via the IBM Business Automation Workflow (BAW). The BAW is used across the ONS for a variety of purposes, so you may have access to this tool already.</p>
+      <p>If you do not have access to the IBM Business Automation Workflow (BAW): </p>
+        {{
+            onsList({
+                "element": 'ol',
+                "itemsList": [
+                    {
+                        "text": 'You can gain access to the BAW via ServiceNow in “I Want Something > 04 – Software, Applications & Platforms > BAW Support”.'
+                    },
+                    {
+                        "text": 'Select the option “Add A New BAW User”.'
+                    },
+                    {
+                        "text": 'Under “Process app name” enter “SML Method Request”.'
+                    },
+                    {
+                        "text": 'Fill out the other information as appropriate to you.'
+                    }
+                ]
+            })
+        }}
+      <p>If you already have access to the BAW, and want access to the SML forms: </p>
+        {{
+          onsList({
+              "element": 'ol',
+              "itemsList": [
+                  {
+                      "text": 'You can gain access to the SML forms via ServiceNow in “I Want Something > 04 – Software, Applications & Platforms > BAW Support”.'
+                  },
+                  {
+                      "text": 'Selection the option “Change A User’s Access To A Process App”.'
+                  },
+                  {
+                      "text": 'Under “Process app name” enter “SML Method Request”.'
+                  },
+                  {
+                      "text": 'Fill out the other information as appropriate to you.'
+                  }
+              ]
+          })
+        }}
+      <p>Once you have been granted access to the SML forms on the BAW, login to the BAW here: 
+        {% from "components/external-link/_macro.njk" import onsExternalLink %}
+            {{
+                onsExternalLink({
+                  "url": 'https://www.bpm.ibmcloud.com/auth/index.jsp',
+                  "linkText": 'https://www.bpm.ibmcloud.com/auth/index.jsp' 
+                })
+            }}
+      </p>
+      <p>Click on “Process Portal”. In the left-hand menu you will see “Create SML Method Request” - if you cannot see a menu on the left-hand side of the screen, click the three horizontal lines at the top left corner of the screen.</p>
+      <p>Click on "Create SML Method Request”, and fill out the information in this form to create the desired request.</p>
+      <h4>How do I submit new methods, or change existing methods, in the SML as a non-ONS user?</h4>
+      <p>Currently we do not accept formal method submissions or change requests from external users, In future, we will accept certain method requests via the Integrated Data Service
```

Comment:
Would it be possible to seperate the content for pages that need full templating functionality into seperate templates?

Reply:
yes i understood .ons design system external links(small icon after the url) is hard to achieve with markdown and Jsonnet. I have looked for alternate approach without design system -  embedding external link icon image in markdown or safe escaping the html content .but these are not nice way & hard to maintain in the long run .